modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManager.java

```diff
@@ -358,6 +363,100 @@ public SCIMResponse listWithGET(UserManager userManager, String filter,
         }
     }
 
+    /**
+     * Build Node for filtering.
+     *
+     * @param filter Filter in the request.
+     * @param schema Schema
+     * @return Node
+     * @throws BadRequestException
+     * @throws IOException
+     */
+    private Node buildNode(String filter, SCIMResourceTypeSchema schema) throws BadRequestException, IOException {
+
+        if (filter != null) {
```

Comment:
use stringUtils

Reply:
apache-commons in not in Charon. Therefore, unable to user StringUtils.